docs/source/zero-code-change-limitations.rst

```diff
@@ -47,6 +47,33 @@ can also visually inspect the resulting cluster assignments.
 ``sklearn.decomposition.TruncatedSVD``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+``sklearn.ensemble.RandomForestClassifier`` / ``sklearn.ensemble.RandomForestRegressor``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The random forest in ``cuml.accel`` uses a different algorithm to find tree node splits.
+When choosing split thresholds, the ``cuml.accel`` random forest considers only quantiles
+as threshold candidates, whereas the scikit-learn random forest considers all possible
+feature values from the training data. As a result, the ``cuml.accel`` random forest
+may choose different split thresholds from the scikit-learn counterpart, leading to
+different tree structure. You can tune the fineness of the quantiles by adjusting the
+``n_bins`` parameter.
```

Comment:
Sklearn doesn't have an `n_bins` parameter, does it? If not, 0cc users won't be able to access it.

Also, can we add a reminder that we expect the quality of results to be the same despite these differences and information on how to measure quality?